packages/input-checkbox/package.json

```diff
@@ -24,21 +21,15 @@
     "url": "https://github.com/CraveFood/farmblocks/issues"
   },
   "homepage": "https://github.com/CraveFood/farmblocks#readme",
-  "keywords": [
-    "farmblocks",
-    "react",
-    "widget",
-    "checkbox",
-    "form",
-    "input"
-  ],
+  "keywords": ["farmblocks", "react", "widget", "checkbox", "form", "input"],
   "peerDependencies": {
     "prop-types": "^15.6.0",
     "react": "^16.0.0",
     "styled-components": "^3.0.2"
   },
   "dependencies": {
     "@crave/farmblocks-hoc-disabled-tooltip": "^1.1.9",
+    "@crave/farmblocks-input-checkbox": "^2.0.0",
```

Comment:
is this correct? `farmblocks-input-checkbox` having `farmblocks-input-checkbox` as a dependency?

Reply:
Nope... I installed it on the wrong package and I thought I had removed... Will do that, thanks